javascript/javascript1/week4/class-exercises/palindromic-substring.js

```diff
@@ -0,0 +1,25 @@
+const isPalindrome = (str) =>
+  str.toLowerCase().split('').reverse().join('') == str.toLowerCase();
+
+function getLongestPalindrome(str) {
+  let longestSubstr = '';
+  //Loop through every characters of string
```

Comment:
I see some minor formatting inconsistencies which could be sorted with running Prettier on file save. This article describes how to do it - https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code#step-2-formatting-code-on-save